src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml

```diff
@@ -115,7 +115,7 @@
                                     </Grid.ColumnDefinitions>
 
                                     <Rectangle Name="Bg" Stroke="Transparent" Fill="Transparent" StrokeThickness="0" Grid.ColumnSpan="5"/>
-                                    <ContentPresenter Grid.Column="0" x:Name="Icon" Margin="22,1,4,1" Width="16" Height="16" VerticalAlignment="Center"
+                                    <ContentPresenter Grid.Column="0" x:Name="Icon" Margin="28,1,4,1" Width="16" Height="16" VerticalAlignment="Center"
```

Comment:
The left value is calculated 
4 (header icon left margin) + 16 (width of header icon) + 4 (header icon right margin) + 4 (Header text block left margin)
= 28

Reply:
Can we try with just 24. Header icon width + margins. That may help address some of the problems with "22" here but avoid adding too much space